src/adagrams.js

```diff
@@ -1,15 +1,108 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterPool = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+  };
+  let copiedPool = JSON.parse(JSON.stringify(letterPool));
+  let chosenLetters = [];
+  while (chosenLetters.length < 10) {
+    const keys = Object.keys(copiedPool);
+    let randomLetter = keys[Math.floor(Math.random() * keys.length)];
+    if (copiedPool[randomLetter] > 0) {
+      chosenLetters.push(randomLetter);
+      copiedPool[randomLetter] -= 1;
+    }
+  }
```

Comment:
This doesn't quite mimic the same probabilities as putting all the tiles in a bag and picking ten tiles.

When the program randomly picks on L34, it's a random pick of the 26 letters in the alphabet. That means A and Z both have 1/26th chance of being picked before we do the validation check, despite the letter bag distribution for A being 9/98 and Z being 1/98.